reference/twig_reference.rst

```diff
@@ -324,6 +324,11 @@ Returns the relative URL (without the scheme and host) for the given route.
 If ``relative`` is enabled, it'll create a path relative to the current
 path. More information in :ref:`templating-pages`.
 
+.. note::
+
+    ``parameters`` accept some special parameters. More information in
+    :ref:`special-routing-parameters`.
```

Comment:
@xabbuh I followed your last advice and then I realised that there is a link to the page you suggested just before the section I added. So I added a link to the Routing guide.